spark-screenshot-testing/src/test/kotlin/com/adevinta/spark/PreviewScreenshotTests.kt

```diff
@@ -67,10 +67,11 @@ internal class PreviewScreenshotTests {
                         else -> name
                     }
                     paparazzi.unsafeUpdateConfig(deviceConfig = BaseDeviceConfig.PIXEL_5.deviceConfig)
-                    paparazzi.snapshotPlus(name = previewNameWithVariants) {
+                    paparazzi.sparkSnapshot(name = previewNameWithVariants) {
                         showkaseComponent.component()
                     }
-                } catch (e: Exception) {
+                } catch (t: Throwable) {
+                    // TODO-@soulcramer (22-48-2023): Should we display the throwable message here too?
```

Comment:
😬 What I told you the other day was in fact wrong.
If we keep these tests in a single `@Test` method we won't be able to make any use of the multiple potential errors...

Reply:
Let's ignore this for now until we have dedicated test methods for each composable. WDYT?

Reply:
Yeah I know but we said that it'll at least until have dedicated components tests to avoid the test to fail on the first screenshot that is not the same.

Reply:
Oh I see. The images will be in the artifact, even though there is only a single test. 👍